packages/fx-core/src/component/driver/teamsApp/copyAppPackageToSPFx.ts

```diff
@@ -32,7 +33,12 @@ export class copyAppPackageToSPFxDriver implements StepDriver {
     spfxFolder: asString,
   });
 
-  @hooks([addStartAndEndTelemetry(actionName, actionName)])
+  @hooks([
```

Comment:
Does it work when adding hooks to run()? I saw that coordinator will call execute() prior to run().

Reply:
Yes, it works with testing 
![image](https://user-images.githubusercontent.com/73154171/230287856-ea5d494e-ff03-4bd9-861d-34f9b0d9af92.png)